packages/mask/src/extension/popups/pages/Wallet/GasSetting/Prior1559GasSetting.tsx

```diff
@@ -87,6 +89,11 @@ const useStyles = makeStyles()((theme) => ({
     },
 }))
 
+const minGasPriceOfChain: { [key in ChainId]?: string } = {
```

Comment:
extract `{ [key in ChainId]?: string }` as shared type?

Reply:
No, we have many of the same types, but the `value` has a different meaning.